tests/run/i4430.scala

```diff
@@ -0,0 +1,5 @@
+object Test {
+  def main(args: Array[String]): Unit = {
+    (new java.lang.StringBuilder()).append(Array[Char](), 0, 0)
```

Comment:
This should be a [DottyBytecodeTests](https://github.com/lampepfl/dotty/blob/master/compiler/test/dotty/tools/backend/jvm/DottyBytecodeTests.scala) that checks the call signature.

Reply:
Can somebody else do the bytecode test? It's one of the few things I don't want to get into.